nix/v/t1-elaborater-jar.nix

```diff
@@ -0,0 +1,110 @@
+{ lib
+, lib'
+, stdenv
+, fetchMillDeps
+, fetchFromGitHub
+, makeWrapper
+, jq
+, jre
+
+  # chisel deps
+, mill
+, espresso
+, circt
+, protobuf
+, antlr4
+
+  # Compile options
+, elaborate-config ?  ../../configs/v1024l8b2-test.json
+}:
+
+let
+  emulator-config-name = lib.removeSuffix ".json" (baseNameOf elaborate-config);
+  submodules = {
```

Comment:
Is dev shell happy to edit the source code from submodules?

Reply:
cp -r should also include the .git folder. But git clone from nix store and git remote set to upstream instead.